core/src/main/java/io/th0rgal/oraxen/utils/BlockHelpers.java

```diff
@@ -172,4 +172,7 @@ public static boolean isLoaded(Location loc) {
         return loc.getWorld() != null && isLoaded(loc.getWorld(), loc);
     }
 
+    public static boolean isRegularNoteBlock(Block block) {
+        return !OraxenBlocks.isOraxenNoteBlock(block) && block.getType() == Material.NOTE_BLOCK;
```

Comment:
isOraxenNoteBlock checks if its a noteblock already

Reply:
isOraxenNoteBlock just checks if its a custom oraxen noteblock, not if it's a regular noteblock, if I remove the block.getType() equals noteblock check it will play a note if u right click on any block except oraxen blocks

Reply:
Move the block-check first then as it is "less taxing"
indeed the check just checks if the blockdata is in the map